src/components/Carousel/utils.ts

```diff
@@ -46,11 +46,14 @@ export const getPosition = (index: number, props: CarouselProps): number => {
  * @param forceReflow
  */
 export const setPosition = (position: number, axis: 'horizontal' | 'vertical'): React.CSSProperties => {
-    const style = {};
```

Comment:
To keep the old implementation, but still remove the @ts-ignore you could just type the `style` local variable as a `React.CSSProperties`